src/draw/handler/Draw.Polygon.js

```diff
@@ -70,13 +79,23 @@ L.Draw.Polygon = L.Draw.Polyline.extend({
 	},
 
 	_getMeasurementString: function () {
-		var area = this._area;
+		var area = this._area,
+			measurementString = '';
+
 
-		if (!area) {
+		if (!area && !this.options.showLength) {
 			return null;
 		}
 
-		return L.GeometryUtil.readableArea(area, this.options.metric);
+		if (this.options.showLength) {
+			measurementString = L.Draw.Polyline.prototype._getMeasurementString.call(this);
+		}
+
+		if (area) {
+			measurementString += '<br>' + L.GeometryUtil.readableArea(area, this.options.metric, this.options.precision);
```

Comment:
Dit is zoals het al was; er was alleen nog de mogelijkheid voor standaard imperial, of metric als het vlaggetje is gezet.